plugins/catalog-backend/src/ingestion/LocationReaders.ts

```diff
@@ -74,10 +76,12 @@ export class LocationReaders implements LocationReader {
         } else if (item.type === 'entity') {
           if (rulesEnforcer.isAllowed(item.entity, item.location)) {
             const entity = await this.handleEntity(item, emit);
-            output.entities.push({
-              entity,
-              location: item.location,
-            });
+            if (entity) {
```

Comment:
there didn't use to be an escape route from handleEntity, but now that it can fail in various ways it needs to be able to return undefined and abort processing